### ISSUE TITLE
train_ota.py

### ISSUE THREAD

**Issue by zzbbzz626** [SSTato/YOLOv7_obb#18]
Custom dataset with very low accuracy after training with train_ota.py, what is the reason for this, and is OTA suitable for rotated object detection?

**Comment by niehandsome** [comment]
I used the train.py file for training without using pre training weights. After 300 rounds of training, the map was only 0.15. Do you understand this reason?

**Comment by SSTato** [comment]
Please provide more information, such as training result graphs, etc

**Comment by niehandsome** [comment]
![result](https://github.com/SSTato/YOLOv7_obb/assets/112600602/0c1936d1-faa8-45b9-a713-7a07c7403923)
This is the training result
![train](https://github.com/SSTato/YOLOv7_obb/assets/112600602/dd7c22e0-dae8-4ca6-a1a5-e0c2516535a6)
This is the parameter setting during training
![YOLOV7](https://github.com/SSTato/YOLOv7_obb/assets/112600602/45aa57b0-571f-41fe-b3d7-e134c7b1dbc3)
This is the model structure file
![data](https://github.com/SSTato/YOLOv7_obb/assets/112600602/0da29709-4a09-486d-8631-1c86480ac572)
This is the dataset settings file
![hdata](https://github.com/SSTato/YOLOv7_obb/assets/112600602/35e93190-c240-4ef4-844a-a85f2253b6ed)
This is a hyperparameter file

**Comment by niehandsome** [comment]
My goal is only one class, and I am trying to add a class of "none" to ensure classification loss, but the dataset can be found but showed corrupted.

**Comment by SSTato** [comment]
If you do not use pre trained models, the convergence speed will decrease, which is closely related to your dataset. From the training result graph, it can be seen that your model has not converged. You should increase the number of epochs to ensure the convergence of your model. You can use this 300 epochs model as a pre training model. Not all datasets are suitable for 300 epochs, you need to analyze them based on specific circumstances, and of course, prevent overfitting. You can use Tensorboard to view your training progress in real-time.
![image](https://github.com/SSTato/YOLOv7_obb/assets/93987794/87a5e74a-dc35-48fc-9b17-f93a5dd3f479)
